### ISSUE TITLE
Packagist links to bitbucket

### ISSUE THREAD

**Issue by philippejadin** [kwi-dk/UrlLinker#1]
Hello,

It seems that packagists links to the bitbucket repository, which causes composer to fails.

Anything changed recently?

Thank you!

**Comment by kwi-dk** [comment]
Yes, faced with Bitbucket ending Mercurial support, the repository was converted from Mercurial to Git and moved to GitHub. I'm not aware of packagists; that's something someone else must have set up.

The latest version on Bitbucket was a work in progress from a contributor that was never quite finished. This is available on the "bleeding" branch here on GitHub, while "master" is the older version that's closer to my original StackOverflow answer.

**Comment by ChristophWurst** [comment]
Are there any plans on republishing this app?

**Comment by ChristophWurst** [comment]
cc @MacDada who is also listed on https://packagist.org/packages/kwi/urllinker

**Comment by kwi-dk** [comment]
I'm not going to, given that I never published it to begin with.

If someone else wants to do so, go ahead, but I'd recommend doing so from your own fork of the repo, to avoid mishaps like this in the future. :)

**Comment by MacDada** [comment]
I’m gonna look at it tomorrow

**Comment by MacDada** [comment]
OK, I [forked it](https://github.com/MacDada/UrlLinker) and changed destination [on packagist](https://packagist.org/packages/kwi/urllinker).
